task/fips-operator-check/0.1/fips-operator-check-step-action.yaml

```diff
@@ -0,0 +1,194 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: StepAction
+metadata:
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: "konflux"
+  name: fips-operator-check-step-action
+spec:
+  description: >-
+    Scans operator bundle image builds for FIPS compliance using the check-payload tool.
+  params:
+    - name: image-digest
+      description: Image digest to scan.
+    - name: image-url
+      description: Image URL.
+  results:
+    - name: TEST_OUTPUT
+      description: Tekton task test output.
+    - name: IMAGES_PROCESSED
+      description: Images processed in the task.
+  image: quay.io/yashn/konflux-test-yashn:latest-amd64
```

Comment:
The image will be updated with konflux-test once https://github.com/konflux-ci/konflux-test/pull/331 is merged

Reply:
Image has been updated. PR ready for review